node/pkg/common/chainlock.go

```diff
@@ -128,11 +138,15 @@ func UnmarshalMessagePublication(data []byte) (*MessagePublication, error) {
 
 	reader := bytes.NewReader(data[:])
 
-	txHash := common.Hash{}
-	if n, err := reader.Read(txHash[:]); err != nil || n != HashLength {
-		return nil, fmt.Errorf("failed to read TxHash [%d]: %w", n, err)
+	txIdLen := uint32(0)
```

Comment:
Should this be `uint8`, since the txID should only ever be 64? Maybe there's some compatibility concerns I'm not aware of.